sphinx/conf.py

```diff
@@ -389,5 +389,6 @@ def getValue(value):
     r'https://trac.openmicroscopy.org*',
     r'http://openslide.cs.cmu.edu*', # NameResolutionError
     r'https://openslide.cs.cmu.edu*', # NameResolutionError
-    r'https://tronche.com/computer-graphics/*' # NameResolutionError
+    r'https://tronche.com/computer-graphics/*', # NameResolutionError
+    r'http://www.bio-rad.com/*' # 403 Client Error: Forbidden for url
```

Comment:
Could you try to use https instead of excluding?

Reply:
fixed in https://github.com/ome/bio-formats-documentation/pull/394/commits/a3ef84ee89ed69228628ee1232a41bb80df45a72